examples/07_multiple_key_join.py

```diff
@@ -51,6 +51,20 @@
 flights = flights.sample(50_000, random_state=1, ignore_index=True)
 flights.head()
 
+###############################################################################
+# Let us see the arrival delay of the flights in the dataset:
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+sns.set_theme(style="ticks")
+
+sns.histplot(data=flights, x="ArrDelay")
```

Comment:
What about setting a log scale for the y-axis? This would make the plot more readable, WDYT?

```suggestion
sns.histplot(data=flights, x="ArrDelay", log_scale=(False, True))
```

![Screen Shot 2023-08-21 at 17 03 13](https://github.com/skrub-data/skrub/assets/18560386/e9976c22-eb2e-4a67-8c01-c6db22b691bc)

The x-axis is skewed as well but no log transformation can be applied because this is non-positive.